buildSrc/build.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
     implementation 'com.google.cloud.tools.jib:com.google.cloud.tools.jib.gradle.plugin:3.2.1'
     implementation 'com.srcclr.gradle:com.srcclr.gradle.gradle.plugin:3.1.12'
     implementation 'de.undercouch.download:de.undercouch.download.gradle.plugin:5.1.0'
-    implementation 'gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.7.2'
+    implementation 'ru.vyarus:gradle-quality-plugin:4.7.0'
```

Comment:
Are there any concerns with using this plugin over spotbugs? It seems like it's maintained by a single person as a personal project. The latest version is 4.8.0.

Reply:
I switched this because I couldn't figure out how to exclude generated code using the other plugin without changing the generated code package. It has the benefit of supporting a few different code analysis tools too. Right now we're running spotbugs but using the most lenient setting, switching it to a different setting would require excluding these files. But this is all separate from fixing sonar errors so I can back this part out and create a separate tech debt ticket for spotbugs.

Reply:
I reverted this change